### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by emielvdveen**

### CODE REVIEW

File: core/assets/js/tabbar.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    updated() {
+        this.show(activeTabId, false)
+    },
+
+    show(nextTabId, scrollToTop) {
+        activeTabId = nextTabId
+        var tabs = Array.from(document.querySelectorAll('[id^="tab_"]'));
+
+        if (tabs.filter((tab) => { return tab.id === nextTabId }).length === 0) {
```
Comment (bliksemman) [inline core/assets/js/tabbar.js:17]: [Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) is a bit nicer than filter + length.

File: core/assets/js/tabbar.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            // skip unknown tab
+            return
+        }
+
+        // Show active tab
+        tabs.forEach(tab => {
+            var tab_id = tab.getAttribute("id")
```
Comment (bliksemman) [inline core/assets/js/tabbar.js:24]: Why `getAttribute` here? The code above used `.id`.

File: core/lib/core_web/ui/navigation/tabbar.ex [hunk 3]
```diff
@@ -2,13 +2,14 @@ defmodule CoreWeb.UI.Navigation.Tabbar do
...
 
+  prop(initial_tab, :integer, required: true)
   prop(size, :atom, default: :wide)
 
   alias CoreWeb.UI.Navigation.{TabbarWide, TabbarNarrow}
 
   def render(assigns) do
     ~H"""
-    <div>
+    <div id="tabbar" initial-tab={{@initial_tab}} phx-hook="Tabbar">
```
Comment (bliksemman) [inline core/lib/core_web/ui/navigation/tabbar.ex:12]: Maybe use a [data attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes)? This avoid collisions with updated specs & has a slightly nicer API.

**Review by bliksemman — APPROVED** [review]

File: core/assets/js/tabbar.js (hunk 2 above)
Comment (emielvdveen) [reply core/assets/js/tabbar.js:24]: No specific reason

File: core/assets/js/tabbar.js (hunk 1 above)
Comment (emielvdveen) [reply core/assets/js/tabbar.js:17]: Changed in my next PR

File: core/assets/js/tabbar.js (hunk 2 above)
Comment (emielvdveen) [reply core/assets/js/tabbar.js:24]: Changed in my next PR

File: core/lib/core_web/ui/navigation/tabbar.ex (hunk 3 above)
Comment (emielvdveen) [reply core/lib/core_web/ui/navigation/tabbar.ex:12]: Changed in my next PR

**Pull request merged**